lib/dockerdaemon/image_pull.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2016-2019 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package dockerdaemon
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+)
+
+// ImagePull calls `docker pull` on an image
+func (cli *dockerClient) ImagePull(ctx context.Context, registry, repo, tag string) error {
```

Comment:
Please don't separate receiver methods from the file in which the type is declared.